integration-tests/tests/snapd_test.go

```diff
@@ -64,6 +65,19 @@ func (s *snapdTestSuite) SetUpTest(c *check.C) {
 
 	s.cmd.Start()
 
+	done := false
+	for i := 0; i < 50; i++ {
+		if err := exec.Command("sudo", "chmod", "/run/snapd.socket", "0666").Run(); err == nil {
```

Comment:
It would be also fine to use the functions at integration-tests/testutils/cli instead of exec.Command so that we get a log entry for the executed command